packages/scrapers-v2/test/urls.test.ts

```diff
@@ -1,10 +1,10 @@
 // https://catalog.northeastern.edu/undergraduate
 
-import { scrapeMajorLinks } from "../src/urls/scrape_urls";
+import { scrapeMajorLinks } from "../src/urls/urls";
 
 describe("url scraper", () => {
   test("scrape", async () => {
-    jest.setTimeout(15000)
+    jest.setTimeout(15000);
```

Comment:
I'm a little confused as to why we need to use a settimeout here. Is it because fetching the major links is async and if it wasn't here, the test function would complete before we get our result?

Reply:
async jest tests have a default limit of 5sec, after which, they auto-fail. this is here because the scraper tests take around ~5s to run (we add some buffer for slow internet connection, etc).

Reply:
https://jestjs.io/docs/jest-object#jestsettimeouttimeout <- mentions the default 5s timeout

Reply:
LOL from looking at the docs we are using this wrong, we should be using this instead: https://jestjs.io/docs/api#testname-fn-timeout -> will fix